query/src/test/resources/configuration-parsing-test.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <infinispan
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-      xsi:schemaLocation="urn:infinispan:config:9.1 http://www.infinispan.org/schemas/infinispan-config-9.1.xsd"
-      xmlns="urn:infinispan:config:9.1">
+      xsi:schemaLocation="urn:infinispan:config:9.2 http://www.infinispan.org/schemas/infinispan-config-9.1.xsd"
```

Comment:
You forgot to bump the actual version in the URL (for this and many other files)

Reply:
fixing

Reply:
probably I did not look at files changed after rebasing...

Reply:
Nope, my search and replace was not ideal :)